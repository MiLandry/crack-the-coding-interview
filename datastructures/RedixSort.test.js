const rs = require('./RedixSort');


test('it sorts', () => {
  expect(rs([4,3,2,1]))
    .toMatchSnapshot()
})

test('it sorts', () => {
  expect(rs([7,1,2,1]))
    .toMatchSnapshot()
})

// test('it sorts', () => {
//   expect(rs([ 170, 45, 75, 90, 802, 24, 2, 66]))
//     .toMatchSnapshot()
// })

