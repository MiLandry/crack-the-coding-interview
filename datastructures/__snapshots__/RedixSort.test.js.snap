// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it sorts 1`] = `
Array [
  1,
  2,
  3,
  4,
]
`;

exports[`it sorts 2`] = `
Array [
  1,
  1,
  2,
  7,
]
`;
